uwreact_bringup/launch/zeds.launch

```diff
@@ -0,0 +1,71 @@
+         RESOLUTION_HD720  -> 1280*720,  available framerates: 15, 30, 60 fps.
+         RESOLUTION_VGA    -> 672*376,   available framerates: 15, 30, 60, 100 fps. -->
+
+    <!-- Definition coordinate frames -->
+    <arg name="publish_tf"                    default="true" />
+    <arg name="publish_map_tf"                default="true" />
+    <arg name="base_frame"                    default="zed_camera_center" />
+
+    <arg name="depth_topic"                   default="depth_registered" />
+    <arg name="depth_cam_info_topic"          default="depth/camera_info" />
+    <arg name="laserscan_topic"               default="laserscan"/>
+
+    <!-- First ZED camera -->
+    <group ns="zed1">
+        <node name="zed_wrapper" pkg="zed_wrapper" type="zed_wrapper_node" output="screen" required="true">
+            <param name="zed_id"                value="$(arg zed_id1)" />
+            <param name="camera_model"          value="0" />
+            <param name="verbose"               value="$(arg verbose)" />
+            
+            <param name="resolution"            value="$(arg resolution)" />
+            <param name="frame_rate"            value="$(arg frame_rate)" />
+            <param name="gain"                  value="100" />
+            <param name="exposure"              value="100" />
+            <param name="auto_exposure"         value="true" />
+            <param name="mat_resize_factor"     value="1.0" />
+            
+            <param name="quality"               value="1" />
+            <param name="sensing_mode"          value="0" />
+            <param name="openni_depth_mode"     value="0" />
+            <param name="confidence"            value="100" />
+            <param name="max_depth"             value="10.0" />
+            <param name="depth_stabilization"   value="1" />
+            
+            <param name="pose_smoothing"        value="false" />
+            <param name="spatial_memory"        value="false" />
+            <param name="floor_alignment"       value="false" />
+            <param name="camera_flip"           value="false" />
+            <param name="publish_tf"            value="$(arg publish_tf)" />
+            <param name="publish_map_tf"        value="$(arg publish_map_tf)" />
+            <param name="init_odom_with_first_valid_pose"   value="true" />
+            <param name="publish_pose_covariance"   value="false" />
+
+            <param name="depth_topic"           value="$(arg depth_topic)" />
+            <param name="depth_cam_info_topic"  value="$(arg depth_cam_info_topic)" />
+            <param name="base_frame"            value="$(arg base_frame)" />
+        </node>
+
+        <node name="depthimage_to_laserscan" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan">
+            <remap from="image"            to="$(arg depth_topic)" />
+            <remap from="camera_info"      to="$(arg depth_cam_info_topic)" />
```

Comment:
the zed node publishes more than one camera_info tho cause it has one for depth, rgb, etc. so will it still know which one to use?

Reply:
I'd try it out to doublecheck, but i think it might not work because theres no camera_info topic thats in the /depth namespace

Reply:
there is

Reply:
The list of topics on the [ros wiki](http://wiki.ros.org/zed-ros-wrapper) does not match the list of topics on [stereolabs' site](http://wiki.ros.org/zed-ros-wrapper). Their site seems to be more up to date, and on their site they list a `/depth/camera_info` topic. I will open an issue to update the wiki. I think we should *not* remap the `camera_info` topic, since it should be handled automatically.

Reply:
It doesn't seem to work unless I map it to the correct caner_info

Reply:
I think this is because you have remapped `/depth/depth_registered` to `/depth_registered`, so the image and info topics are no longer in the same namespace. We should leave the depth_image as its default `/depth/depth_registered`

Reply:
Yep, this fixed it. So we should keep depth_registered in the depth namespace?

Reply:
Yes, I would like to leave as much stuff default as possible, unless we have a valid need to change it. Let's put the depth topic back to its default (`/depth/depth_registered`), and let the image_transport automatically find the camera_info topic.

Thanks!

Reply:
Final nitpick: Rather than specifying the `<param name="depth_topic" value="depth/depth_registered" />` in the `zed_wrapper` node, let's just delete that line and let the wrapper's default take over.